dashboard/app/helpers/survey_results_helper.rb

```diff
@@ -1,5 +1,17 @@
 module SurveyResultsHelper
   def show_diversity_survey?(kind)
+    # Disable diversity survey
+    false
+  end
+
+  def show_nps_survey?(kind)
+    # Disable NPS survey
+    false
+  end
+
+  # For archival purposes, this is the working implementation of
+  # show_diversity_survey? for when the survey is active.
+  def show_diversity_survey_active?(kind)
```

Comment:
It seems simpler and cleaner to simply add `return false` to the top of the existing method, rather than creating a new method for archival purposes. Doing so will necessitate a rubocop disable for unreachable code.